reo/src/load_profile.py

```diff
@@ -581,6 +581,22 @@ def __init__(self, dfm=None, user_profile=None, pvs=[], critical_loads_kw=None,
         doe_reference_name_list = kwargs.pop("doe_reference_name", [])
         self.annual_kwh = kwargs.pop("annual_kwh", None)
 
+        if use_default_outage:
+            print("using default outage")
+            reliability_data = pd.read_csv('reo/src/data/Reliability_2019.csv', header = 1, index_col = 3) 
+            outage_duration = float(reliability_data.loc[outage_utility_name].at['CAIDI With MED'])/60 # CAIDI is given in minutes
+            outage_duration = round(outage_duration * self.time_steps_per_hour)
+            
+            if outage_start_time_step == None:
+                outage_start_time_step = outage_end_time_step - outage_duration  
+            else:
```

Comment:
Might want to check here that `outage_end_time_step` is not `None`, because if it is this will result in an error. If `outage_end_time_step` is `None` it is then you might raise an error or pick a default start time somehow